### PR TITLE
New package: Gifsicle-1.88

### PR DESCRIPTION
**Pull request by ColdPhoenix00**

### CODE REVIEW

File: srcpkgs/gifsicle/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'gifsicle'
+pkgname=gifsicle
+version=1.88
+revision=1
+build_style=gnu-configure
+makedepends="libX11-devel"
+short_desc="Command-line GIF manipulation tool"
+maintainer="ColdPhoenix <coldphoenix@interia.pl>"
+license="GPL-2"
+homepage="https://www.lcdf.org/${pkgname}"
```
Comment (the-maldridge) [inline srcpkgs/gifsicle/template]: Don't put ${pkgname} into the homepage variable, it doesn't get resolved when the update checker writes out the update notifications and so makes the URL useless.

Other than that this looks good.

**Comment by ColdPhoenix00** [comment]
@the-maldridge Fixed hostname variable, as you requested.

File: srcpkgs/gifsicle/template (hunk 1 above)
Comment (ColdPhoenix00) [inline srcpkgs/gifsicle/template]: Fixed hostname variable, as you requested.

**Comment by the-maldridge** [comment]
On further inspection, I think this package should be called 'Gifsicle' but I'm not sure.  @Vaelatern second opinion please.

**Comment by ColdPhoenix00** [comment]
Since @Vaelatern isn't responding, I've changed the package name to Gifsicle.

**Pull request closed without merging**